lighthouse-viewer/app/styles/viewer.css

```diff
@@ -9,7 +9,7 @@
 }
 
 :root {
-  --lh-background-color: #304ffe;
```

Comment:
seems like this isn't used anymore. i think it can die.

Reply:
I'm actually gonna revert it b/c I don't want to touch the viewer at all rn.